src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/UpdateFlowStatusAction.java

```diff
@@ -67,6 +73,27 @@ protected void perform(State from, State to, Event event, FlowUpdateContext cont
         }
     }
 
+    private void saveActionWithDumpToHistory(FlowUpdateFsm stateMachine, Flow flow) {
+        FlowDumpData dumpData = HistoryMapper.INSTANCE.map(flow, flow.getForwardPath(), flow.getReversePath(),
+                DumpType.STATE_AFTER);
+
+        stateMachine.saveActionWithDumpToHistory(format("New %s paths were created", "primary"),
```

Comment:
You don't need format() here if you don't move the format string to the constants.